projects/ngx-scroll-to/package.json

```diff
@@ -59,7 +59,7 @@
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
-    "@angular/common": "11.2.11",
-    "@angular/core": "11.2.11"
+    "@angular/common": "^11.0.0",
+    "@angular/core": "^11.0.0"
```

Comment:
**CONCERN**: Are you sure about depending on a lower version?

Reply:
Yes, it should be compatibile with NG >= 11.0.0 and < 12.